app/helpers/measures_helper.rb

```diff
@@ -2,7 +2,6 @@ module MeasuresHelper
   def filter_duty_expression(measure)
     duty_expression = measure.duty_expression.to_s.html_safe
     duty_expression = '' if duty_expression == 'NIHIL'
-    duty_expression = 'see conditions' if measure.measure_type.id.in? %w[DDA DDJ]
```

Comment:
The measure type ids are excise measure type ids from CHIEF. CDS excise measures are always of the 306 type id

Reply:
CHIEF is gone